File: lap-trinh-c/cod all 24/baitap/Buoi 8/Bai 1.cpp
//8.1
#include <stdio.h>
#include <math.h>

int main(){
	int n,dem=0;
	scanf("%d",&n);
	for(int i=2;i<=sqrt(n);i++){
		if(n%i==0) dem+=2;
	}
	int a=sqrt(n);
	if(pow(a,2)!=n) dem++;
	printf("%d",dem);
	return 0;
}

File: lap-trinh-c/python/bai1.py
import math
a = int(input('a = '))
b = int(input('b = '))
c = int(input('c = '))
delta = math.pow(b,2) - 4*a*c
if delta < 0 :
    print('Phương trình vô nghiệm')
else :
    if delta == 0 :
            x = -b/(2*a)
            print('x = ', x)
    else :

            delta = math.sqrt(delta)
            x1 = (-b+delta)/(2*a)
            x2 = (-b-delta)/(2*a)
            print('x1 = ', x1)
            print('x2 = ', x2)